apps/rush-lib/assets/rush-init/common/config/rush/command-line.json

```diff
@@ -26,7 +26,7 @@
       /**
        * (Required) The name that will be typed as part of the command line.  This is also the name
        * of the "scripts" hook in the project's package.json file.
-       * The name should be comprised of lower case words separated by hyphens.
+       * The name should be comprised of lower case words separated by hyphens or colons.
```

Comment:
Could we add some naming guides?  Something like this:

*The name should be comprised of lower case words separated by hyphens or colons.  The name should include an English verb (e.g. "deploy").  Use a hyphen to separate words (e.g. "upload-docs").  A group of related commands can be prefixed with a colon (e.g. "docs:generate", "docs:deploy", "docs:serve", etc).*

This comment should also go in the **ts-command-line** docs.

Reply:
(BTW I think this convention was popularized by Grunt, [whose command-line](https://adrianmejia.com/blog/2014/10/07/grunt-js-tutorial-from-beginner-to-ninja/#Grunt-tasks) allows something like `grunt example:arg1:arg2 --option=xyz`.  Then `arg1` and `arg2` get passed as special parameters to the task function for `example`, whereas `--option=xyz` is read via a different API.  I point this out because Rush's implementation is a somewhat different design.)

Reply:
Sounds good. I'm not really seeing a good place in the *ts-command-line* README to drop this though, the README doesn't say anything about the allowed characters for actions or parameters, nor the recommendations around their use. The best place I can see to add this would be in the doc comment for the CommandLineAction class.

Reply:
That is what I meant.